code/gbq/tests/test_gbq_qr/plot_prediction.R

```diff
@@ -0,0 +1,27 @@
+# Plot predictions in tests/test_gbq_qr/2024-03-30-UMass-gbq_qr.csv
```

Comment:
Since this function is in a test folder, is there a way to programmatically check whether or not everything works as expected? (I did a cursory look around to see if this function is being called somewhere else but couldn't find anything)